src/components/dashboard/Dashboard.js

```diff
@@ -33,7 +33,7 @@ import ClaimButton from '../common/buttons/ClaimButton'
 import Section from '../common/layout/Section'
 import Wrapper from '../common/layout/Wrapper'
 import logger from '../../lib/logger/pino-logger'
-import { FAQ, PrivacyArticle, PrivacyPolicy, Support, TermsOfUse } from '../webView/webViewInstances'
+import { FAQ, PrivacyArticle, PrivacyPolicy, Statistics, Support, TermsOfUse } from '../webView/webViewInstances'
```

Comment:
General notice: we can move topItems, bottomItems inside useSidemenu hook. it will be logically as the hook should perform everything related to side menu management including providing items lists.
And of course i better to wrap getMenuItems() call to useMemo

Reply:
@serdiukov-o-nordwhale 
Done. I used your patch you share with me and made some fixes - see new commit.